### PR TITLE
Update Summarize Discussion.md

### PR DESCRIPTION
**Pull request by sarikasu**
Wrote instructions since none exist- I did use prompts that were referenced in other Hackman papers

### CODE REVIEW

File: tasks/Summarize Discussion.md [hunk 1]
```diff
@@ -43,6 +43,43 @@ Step Five: Once the second trial was complete, experimenters collected the solut
...
+
+You will be participating in a study on task types, time limits and group performance and given specific tasks to work on as a group. You will need to document your answers to each task and will submit responses at the end of each task session.  
+
+You will be observed by one of our team members throughout the experiment but they will not help you in any way.
+
+The first problem you will need to work on as a group is to discuss an issue, specifically what makes for success in our culture. You will have 10 minutes to work on this task. Please use the full time available. 
+
+After time passes
+
+We will now collect your responses to the discussion problem. You will now have 20 minutes to discuss if birth control pills be made available without prescription.
```
Comment (markwhiting) [inline tasks/Summarize Discussion.md:58]:
```suggestion
We will now collect your responses to the discussion problem. You will now have 20 minutes to discuss if birth control pills should be made available without prescription. 
```

File: tasks/Summarize Discussion.md [hunk 2]
```diff
@@ -43,6 +43,43 @@ Step Five: Once the second trial was complete, experimenters collected the solut
...
+
+You will be participating in a study on task types, time limits and group performance and given specific tasks to work on as a group. You will need to document your answers to each task and will submit responses at the end of each task session.  
+
+You will be observed by one of our team members throughout the experiment but they will not help you in any way.
+
+The first problem you will need to work on as a group is to discuss an issue, specifically what makes for success in our culture. You will have 20 minutes to work on this task. Please use the full time available. 
+
+After time passes
+
+We will now collect your responses to the discussion problem one. You will now have 10 minutes to discuss if birth control pills be made available without prescription.
```
Comment (markwhiting) [inline tasks/Summarize Discussion.md:75]:
```suggestion
We will now collect your responses to the discussion problem one. You will now have 10 minutes to discuss if birth control pills should be made available without prescription. 
```

File: tasks/Summarize Discussion.md [hunk 3]
```diff
@@ -43,6 +43,43 @@ Step Five: Once the second trial was complete, experimenters collected the solut
...
+The first problem you will need to work on as a group is to discuss an issue, specifically what makes for success in our culture. You will have 20 minutes to work on this task. Please use the full time available. 
+
+After time passes
+
+We will now collect your responses to the discussion problem one. You will now have 10 minutes to discuss if birth control pills be made available without prescription. 
+
+After time passes
+
+We will now collect your responses to the discussion problem two. We thank you for your time today. 
+![image](https://user-images.githubusercontent.com/78745728/126358788-d5f63a5e-19d1-4c7e-8441-0f8ec233b2cd.png)
```
Comment (markwhiting) [inline tasks/Summarize Discussion.md:80]: Where's this image from? A paper or did you create it? 

It doesn't add much, right? Its just the same as the text above. If thats the case, perhaps we don't need it?

File: tasks/Summarize Discussion.md [hunk 4]
```diff
@@ -43,6 +43,43 @@ Step Five: Once the second trial was complete, experimenters collected the solut
 
 Step Six: once the questionnaires were completed the participants were debriefed and excused 
 
+## Instructions 
+
```
Comment (markwhiting) [inline tasks/Summarize Discussion.md:47]: If these are instructions you wrote, lets just document that, i.e.,
```suggestion
> Instructions developed by CSSL research team

```

**Review by markwhiting — CHANGES_REQUESTED** [review]
A few minor changes below.

**Review by sarikasu — COMMENTED** [review]
Integrated feedback! So strange about that image, I wrote out instructions in word first so it must have accidentally pasted when I copied them into GitHub

**Review by markwhiting — APPROVED** [review]

**Pull request merged**